app/Models/WpOrg/Plugin.php

```diff
@@ -138,7 +141,7 @@ public static function createFromSyncPlugin(SyncPlugin $syncPlugin): self
 
         DB::beginTransaction();
 
-        $instance = self::create([
```

Comment:
the use of ::query() is dandy for me when it's an actual query, but this is a create() so I'll revert it there (ultimately it's all just magic sugar for the constructor 🤮 )